contrib/endpoints/src/api_manager/config_manager.cc

```diff
@@ -47,6 +47,11 @@ ConfigManager::ConfigManager(
     }
   }
 
+  if (global_context_->server_config()->has_init_service_configs()) {
```

Comment:
Yes,  you may have to set it from api_manager.

So you are saying ConfigManager is not created yet in api_manager?  If so,  we can use separate PR to create it and use it.